contracts/seed/Seed.sol

```diff
@@ -159,6 +159,9 @@ contract Seed {
     */
```

Comment:
pls change comment to  `The amount of seed tokens to buy.`

Reply:
+ in Lock structure pls change `amount` to `seedAmount`

Reply:
Done